src/http_server.py

```diff
@@ -0,0 +1,30 @@
+import logging
+from http.server import SimpleHTTPRequestHandler, ThreadingHTTPServer
+from threading import Thread
+
+logger = logging.getLogger(__name__)
+
+PORT = 8080
+HOSTNAME = "localhost"
```

Comment:
Should we move these variables to environment variables?

And then create a separate configuration within the `configuration.py`, something like `HttpServerConfiguration`. What do you think?